xcp-clipboardd.c

```diff
@@ -329,16 +318,9 @@ static int handleEvents (struct xs_handle *xs, int qemuFd, unsigned int domId) {
   };
 
   for (;;) {
-    for (;;) {
-      syslog(LOG_DEBUG, "polling events...");
-      int ret = poll(fds, 2, -1);
-      if (ret == -1) {
-        if (errno != EINTR) {
-          syslog(LOG_ERR, "poll failed because: %s", strerror(errno));
-          return -1;
-        }
-      } else if (ret > 0)
-        break;
+    if (xcp_poll(fds, 2, -1) <= 0) {
```

Comment:
No. Please take a look at: https://github.com/xcp-ng/xcp-emu-manager/blob/master/src/emu.c#L670

Reply:
More precisely: Timeout is -1, so the condition should be `xcp_poll(fds, 2, -1) != XCP_ERR_OK`. :)

Reply:
Done